python/tskit/drawing.py

```diff
@@ -291,13 +291,20 @@ def __init__(
         tree_x = self.treebox_x_offset
         break_x = self.treebox_x_offset
 
```

Comment:
A bit less code:
```python
breakpoints = ts.breakpoints(as_array=True)
integer_breakpoints = np.all(breakpoints == np.floor(breakpoints))
```

Looks good otherwise.

Reply:
Yeah, I started writing this by using the numpy breakpoints array, but then I wondered if we would ever want to plot a tick label that wasn't at a breakpoint (e.g. evenly spaced axis ticks), so I separated the concept of a label from the actual value dumped into the label. Happy to revert to using the breakpoints array, though.

Reply:
OK, well why not go through the list of labels afterwards then? Interspersing the logic for checking whether they are integers or not with the code for putting the labels into the list makes it more likely that this check will be forgotten later, if we do change the code. So,
```python
integer_breakpoints = all(round(label) == label for _, _, label in xticks)
```

Reply:
Done in latest push, and tests added